views/browse.slim

```diff
@@ -43,9 +43,15 @@ html lang="en"
     main class="viewport-container" style="margin-top: var(--space-x-large);"
       h1 style="margin-top: 0;" Browse `#{list.original_reference}` in call numbers
       
-      p style="margin-bottom: var(--space-medium);"
-        a href=list.previous_url style="margin-right: var(--space-medium);" Previous page
-        a href=list.next_url Next page
+      ul class="pagination"
```

Comment:
I wonder if something like the following might be good. Would let people find pagination in navigation landmarks.

```html
<nav aria-label="previous and next, pagination">
  <a href="">Previous page</a>
  <a href="">Next page</a>
</nav>
```

Reply:
Good idea! I still kept the unordered list, to follow closer to the [pagination example on a11ymatters](https://www.a11ymatters.com/pattern/pagination/).